.github/workflows/generate-pinval.yaml

```diff
@@ -204,17 +212,25 @@ jobs:
       - name: Install Hugo
         run: sudo dpkg -i ${{ runner.temp }}/hugo.deb
 
-      - name: Generate PINVAL reports
+      - name: Generate Hugo content files and build reports
         env:
           RUN_ID: ${{ needs.setup.outputs.run-id }}
           ENV_CODE: ${{ inputs.environment == 'Production bucket' && 'prod' || 'dev' }}
         run: |
           read -ra PINS <<< "${{ inputs.pins }}"
+          NBHD_FLAG=""
+          if [ "${{ inputs.hugo-build-by-nbhd }}" = "true" ]; then
+            NBHD_FLAG="--build-by-neighborhood"
+          fi
+          # Unquoted neighborhood flag is intentional in this call, since we
+          # want this to expand to nothing (rather than an empty string) if
+          # the variable is empty
           python3 scripts/generate_pinval/generate_pinval.py \
             --run-id "$RUN_ID" \
             --township "$TOWNSHIP" \
             --pin "${PINS[@]}" \
-            --environment "$ENV_CODE"
+            --environment "$ENV_CODE" \
+            $NBHD_FLAG
```

Comment:
This makes sense to me. And the reason we don't treat the other flags this way is because of the optional control flow we've establish with `resolve_model_metadata.py`, right?